### DISCUSSION TITLE
Response to preflight request doesn’t pass access control check: No ‘Access-Control-Allow-Origin’ header is present on the requested resource.

### DISCUSSION THREAD

**Discussion by shakeerah** [adonisjs/core#3307] [Help]
Please I need assistance with my Adonis4 Application. Requests are blocked by cors even though set cors origin to *
config/cors.js
`  origin: '*',`

Cors Error
`Access to XMLHttpRequest at ‘https://api-prot.net/Auth/Login’ from origin ‘https://prot.net’ has been blocked by CORS policy: Response to preflight request doesn’t pass access control check: No ‘Access-Control-Allow-Origin’ header is present on the requested resource.`

Please help.

**Comment by wimurk** [comment]
I also got the same problem! Can't connect to any socket

**Reply by shakeerah** [reply]
> Please I need assistance with my Adonis4 Application. Requests are blocked by cors even though set cors origin to * config/cors.js ` origin: '*',`
> 
> Cors Error `Access to XMLHttpRequest at ‘https://api-prot.net/Auth/Login’ from origin ‘https://prot.net’ has been blocked by CORS policy: Response to preflight request doesn’t pass access control check: No ‘Access-Control-Allow-Origin’ header is present on the requested resource.`
> 
> Please help.

For me, there was an error that was causing the request not to return any response. So it gives a cors error when the request timeout.

Also setting the header to '*' in the config/cors.js was a temporary fix.
`headers: '*', `

Hope this helps.
